lib/validators/specValidator.js

```diff
@@ -38,6 +38,8 @@ class SpecValidator {
    * 
    * @param {object} [options.shouldResolveAllOf] Should allOf references be resolved? Default: true
    * 
+   * @param {object} [options.shouldResolveDiscriminator] Should discriminator be resolved? Default: true
```

Comment:
was this not an option in the currently committed code?

Reply:
I missed documenting it in the specValidator.js class. Actually specValidator simply passes these options to specResolver. So in the new commit, I have removed those options from specValidator as we are checking and validating those in the specResolver cosntructor. Hence it makes no sense to do it twice.